src/Containers/EsiConfiguration.php

```diff
@@ -62,6 +62,11 @@ class EsiConfiguration extends AbstractArrayAccess
         // Redis Cache
         'redis_cache_location' => 'tcp://127.0.0.1',
         'redis_cache_prefix'   => 'eseye:',
+
+        // Memcached Cache
+        'memcached_host' => '127.0.0.1',
+        'memcached_port' => '11211',
```

Comment:
Would it not make sense to add a prefix here in the case of the cache being used with other applications etc?

Reply:
Sure, Memcached has no concept of prefix but we can one at key generation.